src/Avalonia.Themes.Default/Expander.xaml

```diff
@@ -10,7 +10,10 @@
   <Style Selector="Expander[ExpandDirection=Down]">
     <Setter Property="Template">
       <ControlTemplate>
-        <Border Background="{TemplateBinding Background}">
+        <Border Background="{TemplateBinding Background}"
+                BorderBrush="{TemplateBinding BorderBrush}"
+                BorderThickness="{TemplateBinding BorderThickness}"
+                CornerRadius="{TemplateBinding CornerRadius}">
```

Comment:
In some places we have missing BorderBrush/BorderThickness as well. I added them where noticed.